.shellcheckrc

```diff
@@ -2,4 +2,4 @@
 #
 # SC1091: Not following <file>
 # https://github.com/koalaman/shellcheck/wiki/SC1091
-# disable=SC1091
+disable=SC1091
```

Comment:
Why disabling this check ?

Reply:
The check is to validate that something exists on the filesystem before sourcing it, but this project by nature sources things that are dynamically generated.

Reply:
But it was passing before, no? Is there something in this PR that was not passing ?